http/query_handler.go

```diff
@@ -165,10 +165,9 @@ func (h *FluxHandler) handleQuery(w http.ResponseWriter, r *http.Request) {
 		if cw.Count() == 0 {
 			// Only record the error headers IFF nothing has been written to w.
 			err := &influxdb.Error{
-				Code: influxdb.EInternal,
-				Msg:  "failed to execute query against proxy service",
-				Op:   op,
-				Err:  err,
+				Msg: "failed to execute query against proxy service",
+				Op:  op,
+				Err: err,
```

Comment:
I suppose that `HandleHttpError` extracts the error code form `err`, right?

Reply:
right